screens/HomeScreen.tsx

```diff
@@ -1,24 +1,99 @@
-import * as React from 'react';
-import { StyleSheet } from 'react-native';
-import { Text, View } from '../components/Themed';
+import React, { useState } from 'react';
+import * as Animatable from 'react-native-animatable';
+
+import { ScrollView, StyleSheet } from 'react-native';
+import { HeaderText, View } from '../components/Themed';
 import Colors from '../constants/Colors';
+import useColorScheme from '../hooks/useColorScheme';
+import Accordion from 'react-native-collapsible/Accordion';
+import { getHomeScreenSections } from '../services/DataService';
+import { HomeScreenSection } from '../models/HomeScreenSection';
+
+import UilAngleDown from '@iconscout/react-native-unicons/icons/uil-angle-down';
+import UilAngleRight from '@iconscout/react-native-unicons/icons/uil-angle-right';
 
 const HomeScreen: React.FC = () => {
-  return (
-    <View style={styles.container} darkColor={Colors.dark.background}>
-      <Text style={styles.title}>Tab One</Text>
+  const [activeSections, setActiveSections] = useState<number[] | string[]>([]);
+  const colorScheme = useColorScheme();
+
+  const renderHeader = (content: HomeScreenSection, _, isActive: boolean) => {
+    return (
+      <Animatable.View duration={400} transition="backgroundColor">
+        <View style={{ flex: 1, flexDirection: 'column' }}>
+          <View style={{ flex: 1, flexDirection: 'row' }}>
+            <View style={{ flex: 1 }}>
+              <HeaderText style={styles.eventH3}>{content.title}</HeaderText>
+            </View>
+            <View>{renderCollapsibleIcon(isActive)}</View>
+          </View>
+        </View>
+      </Animatable.View>
+    );
+  };
+
+  // :TODO: duplicated in DayScreen.tsx. Abstract?
+  const renderCollapsibleIcon = (isActive: boolean) => {
+    if (isActive) return <UilAngleDown size="26" color="#000000" />;
```

Comment:
Kan je deze evt. vervangen met `<MaterialCommunityIcons name="chevron-down" size={26} color="#000000" />`? Ziet er hetzelfde uit, maar die package is al inbegrepen ih project, hebben we minder dependencies :)

Reply:
Ik dacht dat jij dat al had gedaan op de DayScreen. Ik heb het terug van daar gekopieerd.. :)

Reply:
Aha, yes, dat stond op die map branch denk ik :p Ik zal het fixen